src/state/apis/swappers/helpers/getInputOutputRatioFromQuote.ts

```diff
@@ -186,5 +186,5 @@ export const getInputOutputRatioFromQuote = ({
     .plus(sellSideNetworkFeeUsdPrecision)
     .plus(sellSideProtocolFeeUsdPrecision)
 
-  return netSendAmountUsdPrecision.div(netReceiveAmountUsdPrecision).toNumber()
```

Comment:
fixed incorrect quote ratio calculation such that bad quotes were ranked higher than good ones